storage/mysql/tree_storage_test.go

```diff
@@ -242,115 +283,142 @@ func TestDequeueLeaves(t *testing.T) {
 	db := prepareTestDB(logID, t)
 	defer db.Close()
 	s := prepareTestStorage(logID, t)
-	tx := beginTx(s, t)
-
-	leaves := createTestLeaves(leavesToInsert, 20)
 
-	if err := tx.QueueLeaves(leaves); err != nil {
-		t.Fatalf("Failed to queue leaves: %v", err)
-	}
+	{
+		tx := beginTx(s, t)
+		defer failIfTXStillOpen(t, "TestDequeLeaves", tx)
```

Comment:
Strange. It looks OK in the editor and gofmt doesn't alter it. Same for other cases.